tests/test_dual_debounce_boundaries.py

```
@@ -65,6 +65,19 @@
 from astrbot_plugin_angel_heart.models.analysis_result import SecretaryDecision
 
 
+class FakeClock:
+    """可推进的假时钟，注入 WorkLedger(time_func=...) 构造超时状态。"""
```

Ruff (0.16.1)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)